Modules/Package.swift

```diff
@@ -50,6 +50,7 @@ let package = Package(
         .package(url: "https://github.com/Automattic/wordpress-rs", revision: "alpha-20241116"),
         .package(url: "https://github.com/wordpress-mobile/GutenbergKit", revision: "cc52214a50893b41898607ac0bff7f2787b085bb"),
         .package(url: "https://github.com/Automattic/color-studio", branch: "trunk"),
+        .package(url: "https://github.com/wordpress-mobile/AztecEditor-iOS", revision: "ea5f9aa13e7490647c8ff5c0aac7f9c25614a8df"),
```

Comment:
Change commit to a releas tag?

Reply:
Released https://github.com/wordpress-mobile/AztecEditor-iOS/releases/tag/1.20.0 and updated in [1780f9b](https://github.com/wordpress-mobile/WordPress-iOS/pull/23958/commits/1780f9bc60d635f2a4ea39e07d8622dc732be707)